ep/build-velox/src/setup-centos8.sh

```diff
@@ -59,6 +59,7 @@ function install_build_prerequisites {
   dnf_install ninja-build curl ccache gcc-toolset-11 git wget which
   dnf_install yasm
   dnf_install autoconf automake python39 python39-devel python39-pip libtool
```

Comment:
centos8 does not provide the updates for tzdata, so we used the package for almalinux